src/modules/layout/components/HeaderDropDown.js

```diff
@@ -6,7 +6,7 @@ import style from './HeaderDropDown.css'
 import onClickOutside from 'react-onclickoutside'
 
 type PropsType = {
-  children: React.Element<any>, // fixme: Probably this type is wrong
```

Comment:
Is children a Node ar an Element? Above it is a Node and I think it actually is a Node.

Reply:
We need Element here because 'React.cloneElement()' only works with elements.